modules/ROOT/pages/managing-users.adoc

```diff
@@ -91,6 +91,68 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Rotate a SAML Key
+
+As a security best practice, many organizations rotate the key used to encrypt SAML 2.0 assertions. With the Key Rotation feature, Anypoint Platform can generate a new key, or you can upload a public and private key pair (up to 3).
+
+After you've added a new key to Anypoint Platform, you must ensure the key also exists in your IdP. All keys present in Anypoint Platform are used to decrypt SAML 2.0 assertions sent by your IdP. However, only the primary key is used by Anypoint Platform to sign SAML 2.0 AuthnRequests and SLO requests; the primary key must be configured in your IdP to validate the signatures.
+
+If you are migrating from the default Anypoint Platform SSO certificate to a new certificate, you must update the Assertion Consumer Service (ACS) URL in your IdP. Only the new ACS URL supports the keys generated by the key rotation feature. If your ACS URL already follows the pattern of `.../accounts/login/:org-domain/providers/:providerId/receive-id`, you do not need to change the ACS URL.
+
+=== Generate a Key
+
+Anypoint Platform can create a key for you to use with your IdP.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Click *+ New key*, and select *Generate*. +
+A newly generated key appears in the list of keys.
+
+=== Upload a Key
+
+You can upload an existing public and private key pair.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Click *+ New key*, and select *Upload*.
+. In the *Private key* field, paste your private key.
+. In the *Public key* field, paste your certificate.
+. Click *Upload*. +
+The key you uploaded appears in the list of keys.
```

Comment:
The KEY you uploaded... or the private-public pair or keys?
This is confusing: we introduce a PAIR of keys (private and public) and then say there can be three such pairs, which is SIX keys...and this does not even include the whole "primary" key thing.
I guess I just don't get it.

Reply:
The key "pair" is made of a private key and a public key, also known as a certificate. They make up one key.